web/src/styles/kbd.css

```
@@ -0,0 +1,178 @@
+
+.kbd-md {
+  font-size: 11px;
+  padding: 2px 5px;
+}
+
+/* Inverse variant for tinted surfaces (primary buttons). Sits cleanly on
+   a saturated background without vibrating. */
+.kbd-inverse {
+  background: rgba(255, 255, 255, 0.22);
+  border-color: rgba(255, 255, 255, 0.28);
+  color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)